.travis.yml

```diff
@@ -1,43 +1,20 @@
+language: php
+php:
+  - '7.1'
+  - '7.2'
+  - '7.3'
+  - '7.4'
+  - '8.0'
+jobs:
+  allow_failures:
+    - php: 8.0
+
 before_script:
   - composer install
 
-script: vendor/phpunit/phpunit/phpunit tests/ --whitelist src/ --coverage-clover build/coverage/xml
+script: composer run test
 
 after_script:
   - php vendor/bin/codacycoverage clover build/coverage/xml
 
-_install_windows: &install_windows
-  install:
-    - choco install php --version $PHP_VERSION --no-progress --package-parameters='"/InstallDir:C:\tools\PHP"'
-    - choco install composer --no-progress --ia '"/DEV=C:\tools\php /PHP=C:\tools\php"'
-    - export PATH=$PATH:/C/tools/php
-
-_install_osx: &install_osx
-  install:
-    - brew unlink python@2
-    - brew install php@$PHP_VERSION
-    - brew install composer
-
-jobs:
-  include:
-    - os: linux
-      language: php
-      php: 7.1
-    - os: linux
-      language: php
-      php: 7.2
-    - os: linux
-      language: php
-      php: 7.3
-    - os: osx
-      language: sh
-      env: PHP_VERSION='7.2'
-      <<: *install_osx
-    - os: osx
-      language: sh
-      env: PHP_VERSION='7.3'
-      <<: *install_osx
-    - os: windows
-      language: sh
-      env: PHP_VERSION='7.3.9'
-      <<: *install_windows
+os: linux
```

Comment:
Why did you remove the configuration to run the tests on Windows and OSX?
This package's tests generate files and write them to the file system so it's important to test that this works on all popular operating systems.
I have had issues before with people complaining about the package breaking for windows/osx so it's important to keep these tests intact.
Here's a sample from the travis run for the latest build vs this pull request:
https://travis-ci.org/github/mghoneimy/php-graphql-oqm/builds/721384213
https://travis-ci.org/github/mghoneimy/php-graphql-oqm/builds/754109337

Reply:
I looked into this from a similar PR made on the other repo you have for graphql. 

I can try to make changes, however I'd appreciate if you can suggest a valid configuration for this use case so we can merge this?

Reply:
@mghoneimy bump :)

Reply:
The other package is different, operating system doesn't really make a difference.
For this package the operating system is important for the nature of the tests that run.
I don't understand why can't we just keep the existing configuration?

Reply:
@mghoneimy  
We probably can, I just had trouble figuring out how to run it then took idea from that other repo. 
Would it be too much to ask you to look into this and adjust the config so we can merge this? :)

Reply:
Can you revert these lines to their original and let's see what happens. If travis has a problem I'll look into it

Reply:
I reverted and added what I believe should be code to test PHP8 as well...probably some tweaks needed

Reply:
@mghoneimy any luck with this?

Reply:
Taking a look now, it doesn't make sense that all ios builds are failing, even the old PHP7.2 and PHP7.3 versions

Reply:
Turns out that the failing builds on osx have nothing to do with your change, I restarted the build on `master` without making any changes, and the same build that passed the last time it ran (6 months ago) has failed now.

Seems like travis have done some changes to the way osx jobs run or some changes happened to the package repos.

So we now have 2 options:
1. Fix problem on `master` then rebase this pull request and merge it (my preferred option)
2. Merge this pull request then find a fix to the OSX problem

Reply:
I'd prefer to merge this PR first, then fix the items, as it would greatly improve my current setup. If you have nothing against that, I'd appreciate it.